### DISCUSSION TITLE
need help: how to find specific word on comma separated field.

### DISCUSSION THREAD

**Discussion by a4lmamun** [prisma/prisma#19178] [Q&A]
One of my table has field name tags which stores comma separated string values like "bangladesh,china,friendship", another value "bangladesh-bank,national-security".

I am also using @map field on schema.prisma file as Tags. If i use contains on query where: { Tags: {contains: "bangladesh"} } both fields will show as result. But I only want to get only first one which contains only bangladesh word which is exact match.

There is also a question there where referred to use raw query. By using raw query how can I convert it to match schema.prisma file field name. As I am developing GraphQL API using nodejs. Is there any way to use find_in_set or in any other way to find a solution.

Can anyone suggest me how to get the first value on where query searching for bangladesh tag.

schema.prisma
```prisma
model Content {
    ......
    Tags              String?   @map("tags") @db.Text

    Category    Category    @relation(fields: [CategoryId], references: [Id])
    Subcategory Subcategory @relation(fields: [SubcategoryId], references: [Id])

    @@map("contents")
    @@schema("dp_news")
}
```

db.ts
`export const RequestTagContents = async (tag: string, take: number, skip: number) => {
    return await xprisma.content.findMany({
		where: {
			Tags: {
				...NEED HELP HERE....
			},
                         .....
		},
		include: {
			Category: true,
			Subcategory: true,
		},
	});
};`

**Comment by nurul3101** [comment]
Hello @a4lmamun 👋 

Can you try using [equals](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#equals-1) instead of contains to do an exact match?

**Reply by a4lmamun** [reply]
No i can't use equal as tags field is comma separated string where multiple tags can be stored. I am looking for "bangladesh" tag where ever it stored. It can be findable on "bangladesh,china,friendship" or "bangladesh" or "india,bangladesh" but not in "bangladesh-bank,national-security".

**Reply by nurul3101** [reply]
Oh I see, For now I think you would need to use a [raw sql query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access). You can use [FIND_IN_SET](https://www.w3schools.com/sql/func_mysql_find_in_set.asp) for this case.

**Reply by nurul3101** [reply]
@a4lmamun If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by a4lmamun** [reply]
I would like to get find_in_set feature in prisma. In the meantime i had to switch to another orm to find proper solution.

**Reply by nurul3101** [reply]
Do you mind opening a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for find_in_set?

**Reply by Allakazan** [reply]
@nurul3101 Hey ! Do you know if Prisma supports find_in_set right now ?
